partcad/src/partcad/part_factory_feature_ai.py

```
@@ -112,6 +114,28 @@ def on_init_ai(self):
         self.instantiate_orig = self.instantiate
         self.instantiate = self._instantiate_ai
 
+    def _add_header(self, script):
```

Ruff (0.8.2)

Missing return type annotation for private function `_add_header`

(ANN202)